docs/data/toolpad/core/components/account/account.md

```diff
@@ -8,6 +8,10 @@ components: Account
 
 <p class="description">A component that renders an account management dropdown for your application.</p>
 
+:::info
+If this is your first time using Toolpad Core, you might want to start with the [base concepts](/toolpad/core/introduction/base-concepts/) instead.
```

Comment:
"you might want to" feels a bit too informal and doesn't hold a reason. Maybe we can formulate this more as a recommendation?

```suggestion
If this is your first time using Toolpad Core, we recommend you to read the [basic concepts](/toolpad/core/introduction/base-concepts/) first.
```